services/intersection-api/README.md

```diff
@@ -0,0 +1,152 @@
+# CVManager Intersection API
+
+The cvmanager intersection-api is built off of the [conflictvisualizer api](https://github.com/usdot-jpo-ode/jpo-conflictvisualizer/tree/cvmgr-cimms-integration/api). This directory contains that customized api.
+
+This application is fully dockerized and is designed to run alongside an instance of the [jpo-ode](https://github.com/usdot-jpo-ode/jpo-ode), [jpo-geojsonconverter](https://github.com/usdot-jpo-ode/jpo-geojsonconverter), [jpo-conflictmonitor](https://github.com/usdot-jpo-ode/jpo-conflictmonitor). This project imports pre-built images for these services. If you would like to build them locally, information is available in their respective repositories.
```

Comment:
question: prebuilt images from where? Is this part of the repository, downloading from official release packages, or submodules?

Reply:
These come from the official usdot dockerhub images: https://hub.docker.com/u/usdotjpoode
I added this link in the documentation